boards/nordic/nrf51dk/src/main.rs

```diff
@@ -336,14 +338,19 @@ pub unsafe fn reset_handler() {
     );
     kernel::hil::sensors::TemperatureDriver::set_client(&nrf5x::temperature::TEMP, temp);
 
+    let etor = static_init!(
+        capsules::rng::Entropy32ToRandom<'static>,
+        capsules::rng::Entropy32ToRandom::new(&nrf5x::trng::TRNG)
+    );
     let rng = static_init!(
-        capsules::rng::SimpleRng<'static, nrf5x::trng::Trng>,
-        capsules::rng::SimpleRng::new(
-            &mut nrf5x::trng::TRNG,
+        capsules::rng::RngDriver<'static>,
+        capsules::rng::RngDriver::new(
+            etor,
```

Comment:
Does etor stand for entropy_to_random? I think it is better to use the full name here for clarity!